packages/client/wallets/aa/src/blockchain/wallets/CustomBridge.ts

```diff
@@ -0,0 +1,186 @@
+                if (!this.signer) {
+                    throwUnsupported("eth_signTransaction requires a signer");
+                }
+                const transaction = params![0];
+                transaction.value = `0x${transaction.value.toString(16)}`;
+                transaction.maxPriorityFeePerGas = `0x${transaction.maxPriorityFeePerGas.toString(16)}`;
+                transaction.maxFeePerGas = `0x${transaction.maxFeePerGas.toString(16)}`;
+                transaction.gasLimit = `0x${transaction.gas.toString(16)}`;
+                transaction.chainId = transaction.chainId;
+                transaction.type = 2;
+                delete transaction.gas;
+                // Sign the transaction with the signer
+                const signedTransaction = await this.signer.signTransaction(transaction);
+                // Return the serialized, signed transaction
+                return signedTransaction;
+            }
+
+           case "personal_sign": {
+               if (!this.signer) {
+                   throwUnsupported("personal_sign requires a signer");
+               }
+
+               const message = params![1];
+               const prefixedMessage = `\x19Ethereum Signed Message:\n${message.length}${message}`;
+               const hashedMessage = ethers.utils.keccak256(ethers.utils.toUtf8Bytes(prefixedMessage));
+               return await this.signer.signMessage(ethers.utils.arrayify(hashedMessage));
+           }
+
+            case "eth_sendTransaction": {
+                if (!this.signer) {
+                    return throwUnsupported("eth_sign requires an account");
+                }
+                delete params![0].from;
+                delete params![0].gas;
+                const req = ethers.providers.JsonRpcProvider.hexlifyTransaction(params![0]);
+                req.gasLimit = req.gas;
+                delete req.gas;
+                const tx = await this.signer.sendTransaction(req);
+                return tx.hash;
+            }
+
+            case "eth_getUncleCountByBlockHash":
+            case "eth_getUncleCountByBlockNumber":
+            {
+                coerce = ethers.utils.hexValue;
+                break;
+            }
+
+            case "eth_getTransactionByBlockHashAndIndex":
+            case "eth_getTransactionByBlockNumberAndIndex":
+            case "eth_getUncleByBlockHashAndIndex":
+            case "eth_getUncleByBlockNumberAndIndex":
+            case "eth_newFilter":
+            case "eth_newBlockFilter":
+            case "eth_newPendingTransactionFilter":
+            case "eth_uninstallFilter":
+            case "eth_getFilterChanges":
+            case "eth_getFilterLogs":
+            case "eth_getLogs":
+                break;
+        }
+
+        // If our provider supports send, maybe it can do a better job?
+        if ((<any>(this.provider)).send) {
+            const result = await (<any>(this.provider)).send(method, params);
```

Comment:
the `params` error should be resolved with `params ?? []`
```suggestion
            const result = await this.provider.send(method, params ?? []);
```